### DISCUSSION TITLE
Displaying text in Bangla

### DISCUSSION THREAD

**Discussion by ghost** [wezterm/wezterm#1332] [Q&A]
How should i configure Wezterm to be able to view text in Bangla with proper rendering/layout?

**Comment by wez** [comment]
Is that a language/script with right-to-left layout? wezterm doesn't currently support right-to-left languages

**Comment by ghost** [comment]
> Is that a language/script with right-to-left layout?

No, [Bangla](https://en.m.wikipedia.org/wiki/Bengali_language) is left to right.

**Reply by ghost** [reply]
Paste some text from https://bn.m.wikipedia.org/wiki/বাংলা_ভাষা in wezterm, you will see what's the difference.

**Reply by wez** [reply]
I've opened https://github.com/wez/wezterm/issues/1333 to track the work that needs to be done
